src/components/InlineCodeBlock/WrappedText.js

```diff
@@ -0,0 +1,87 @@
+import React, {Fragment} from 'react';
+import {Text, View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+
+/**
+ * Breaks the text into matrix
+ * for eg: My Name  is Rajat
+ *  [
+ *    [My,'',Name,'','',is,'',Rajat],
+ *  ]
+ *
+ * @param {String} text
+ * @returns {Array<String[]>}
+ */
+function getTextMatrix(text) {
+    return text.split('\n').map(row => row.split(/(\s)/));
+}
+
+const propTypes = {
+    // Required text
+    children: PropTypes.string.isRequired,
+
+    // Style to be applied to Text
+    // eslint-disable-next-line react/forbid-prop-types
+    textStyle: PropTypes.object,
+
+    // Style for each word(Token) in the text,
+    // remember that token also includes the following spaces before next word break
+    // eslint-disable-next-line react/forbid-prop-types
+    wordStyle: PropTypes.object,
+
+    // Style for first word
+    // eslint-disable-next-line react/forbid-prop-types
+    firstWordStyle: PropTypes.object,
```

Comment:
Should these be props or "baked in"? I trust your judgement here, but I just wonder if these props will always be the same, should we use the styles directly instead of passing them as a prop?

Reply:
Yeah. Currently, I don't see any other use cases for these so we can just use them inside the component.